admin/database/database.go

```diff
@@ -55,6 +55,19 @@ type DB interface {
 	CreateProject(ctx context.Context, orgID string, name string, description string) (*Project, error)
 	UpdateProject(ctx context.Context, id string, description string) (*Project, error)
 	DeleteProject(ctx context.Context, id string) error
+
+	// InsertAuthCode inserts the authorization code data into the store.
+	InsertAuthCode(ctx context.Context, code *AuthCode) error
+	// GetByDeviceCode retrieves the authorization code data from the store
+	GetByDeviceCode(ctx context.Context, deviceCode string) (*AuthCode, error)
+	// GetByUserCode retrieves the authorization code data from the store
+	GetByUserCode(ctx context.Context, userCode string) (*AuthCode, error)
+	// UpdateAuthCodeByUserCode updates the authorization code data in the store
+	UpdateAuthCodeByUserCode(ctx context.Context, userCode string, code *AuthCode) error
+	// UpdateAuthCodeByDeviceCode updates the authorization code data in the store
+	UpdateAuthCodeByDeviceCode(ctx context.Context, deviceCode string, code *AuthCode) error
+	// InsertAccessToken inserts the access token data into the store.
+	InsertAccessToken(ctx context.Context, accessToken string, userID string, expires time.Time) error
```

Comment:
ok will use, why do we need `displayName` in user auth token table - https://github.com/rilldata/rill-developer/blob/main/admin/database/database.go#L122, seems redundant  or to lookup in user table why displaying  user tokens ?

Reply:
Display name is not required, it's just so you can add a short description to a token. For browser auth, we set display name to "Browser session". It overlaps slightly with `auth_client_id`, which also has a name, but IMO it doesn't hurt to be able to add extra free text description. (Especially when we open up for manually issuing API tokens)

Reply:
So we don't get displayName from the claims, so what should be inserted when we insert the auth token in db ? If we want to insert displayName for the user confirming the code then we will need to pass it as query parameter in confirmation code, save it in the session and then get it from the session when generating access token during the get token call from the cli.

Reply:
For CLI, if it could just be "CLI login" (no dynamic values). If it's not easy to set, it can just be empty (it will still show the CLI client ID name anyway, so probably doesn't matter).

Reply:
ok I see now, I was thinking you meant to store user displayName here that is present in user table

Reply:
we will need to query the database for getting displayName using the clientId

Reply:
Adding `CLI login`